src/pages/edge/lifecycle-for-edge-network/behavior-reference.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Mobile Core Lifecycle for Edge Network behavior reference
+description: A reference for understanding how Lifecycle for Edge Network behaves under different scenarios.
+keywords:
+- Lifecycle
+- Lifecycle V2
+- Lifecycle for Mobile Core
+- Lifecycle for Platform
+- Lifecycle for Edge Network
+- Lifecycle Edge
+- Lifecycle iOS
+- Lifecycle Android
+- Product reference
+---
+
+# Lifecycle for Edge Network behavior reference
+
+The Lifecycle for Edge Network extension introduces a new way of collecting Lifecycle data and tracking sessions. Unlike the previous Lifecycle extension, which used a configurable timeout window for session tracking, this extension emits Application Launch and Close events, treating each pair as an independent session.
+
+## General behavior
+
+The Lifecycle for Edge Network extension relies solely on the Lifecycle events dispatched by the [`lifecycleStart`](../../home/base/mobile-core/lifecycle/api-reference.md#lifecyclestart) and [`lifecyclePause`](../../home/base/mobile-core/lifecycle/api-reference.md#lifecyclepause) calls to determine the start and end of new sessions and to calculate lifecycle data.
+
+A Lifecycle session is the time between a `lifecycleStart` call and a `lifecyclePause` call. A new session is detected when `lifecycleStart` is called after app first launch or after a `lifecyclePause`.
+
+All events that are not Lifecycle start or Lifecycle pause do not affect Lifecycle state or data. Starting MobileCore using `registerExtensions` does not automatically start a Lifecycle session nor does it affect Lifecycle data.
+
+#### Event reference
+
+Events dispatched by the Lifecycle extension to support Adobe Experience Platform Edge Network workflows include:
+
+* [Lifecycle Application Launch (Foreground)](../../home/base/mobile-core/lifecycle/event-reference.md#lifecycle-application-foreground)
+* [Lifecycle Application Close (Background)](../../home/base/mobile-core/lifecycle/event-reference.md#lifecycle-application-background)
+
+For full event details, please read the [Lifecycle event reference](../../home/base/mobile-core/lifecycle/event-reference.md).
+
+#### Data payloads
+
+The Lifecycle for Edge Network event payloads are defined by the Platform Mobile Lifecycle Details XDM field group, and include information about the application, device, and environment when the event occurred. Please refer to [Lifecycle for Edge Network data](metrics.md) for the data included with these events.
+
+As a consequence of this shift in not relying on a session timeout window to consider new sessions, the potential Lifecycle states are greatly simplified:
+
+### Consecutive Lifecycle API calls
```

Comment:
The heading size here is confusing.  
Is Consecutive Lifecycle API calls under General behavior?

Why it is 
##
####
###?

For 
"As a consequence of this shift in not relying on a session timeout window to consider new sessions, the potential Lifecycle states are greatly simplified:" 
Is it part of Data payloads?  I can't tell by reading with the headings.